pkg/storage/tableland/common/sqlfactory.go

```diff
@@ -71,7 +70,12 @@ func (s *SQLFactory) SQLForInsertingRigAttributes(rigAttrTable string, rigs []lo
 		attVales = append(
 			attVales,
 			goqu.Vals{rig.ID, "string", "VIN", rig.VIN},
-			goqu.Vals{rig.ID, "number", "% Original", math.Round((rig.PercentOriginal90*100)*100) / 100},
+			goqu.Vals{
+				rig.ID,
+				"number",
+				"% Original",
+				strings.TrimRight(strings.TrimRight(fmt.Sprintf("%.2f", rig.PercentOriginal90*100), "0"), "."),
```

Comment:
Here I'm formatting the `float64` value as a string, using up to two decimal places, but removing any trailing `0`s.